dttools/src/Makefile

```diff
@@ -139,6 +140,9 @@ auth_globus.o: auth_globus.c
 auth_test: auth_test.o libdttools.a
 	$(CCTOOLS_LD) -o $@ $(CCTOOLS_INTERNAL_LDFLAGS) $(LOCAL_LDFLAGS) $^ $(LOCAL_LINKAGE) $(CCTOOLS_GLOBUS_LDFLAGS) $(CCTOOLS_EXTERNAL_LINKAGE)
 
+libforce_halt_enospc.so: libforce_halt_enospc.o
+	$(CCTOOLS_CC) -shared -ldl -fPIC $^ -o $@
```

Comment:
-ldl should be at the end of the command, like:

$(CCTOOLS_CC) -shared -fPIC $^ -o $@ -ldl

Otherwise ubuntu complains. Also, a kitty cries every time I have to rewrite a comment because the first time it was ignored. Why do you want kitties to cry?